common/containers/Tabs/GenerateWallet/components/Mnemonic/Word.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import classnames from 'classnames';
+import { translateRaw } from 'translations';
+import './Word.scss';
+
+interface Props {
+  index: number;
+  word: string;
+  value: string;
+  isReadOnly: boolean;
+  onChange(index: number, value: string): void;
+}
+
+interface State {
+  isShowingWord: boolean;
+}
+
+export default class MnemonicWord extends React.Component<Props, State> {
+  public state = {
+    isShowingWord: false
+  };
+
+  public render() {
+    const { index, word, value, isReadOnly } = this.props;
+    const { isShowingWord } = this.state;
+    const readOnly = isReadOnly || isShowingWord;
+
+    return (
+      <div className="MnemonicWord">
+        <span className="MnemonicWord-number">{index + 1}.</span>
+        <div className="MnemonicWord-word input-group">
+          <input
+            className={classnames({
+              'MnemonicWord-word-input': true,
```

Comment:
This can be shortened to 
```ts
classnames(
               'MnemonicWord-word-input',
               'form-control',
               {'is-valid': word === value}
             )
```

Reply:
Or using the same style as LN 49 to keep it consistent